milli/src/update/settings.rs

```diff
@@ -89,6 +89,9 @@ pub struct Settings<'a, 't, 'u, 'i> {
     distinct_field: Setting<String>,
     synonyms: Setting<HashMap<String, Vec<String>>>,
     primary_key: Setting<String>,
+    authorize_typos: Setting<bool>,
+    min_2_typos_word_len: Setting<u8>,
+    min_1_typo_word_len: Setting<u8>,
```

Comment:
Could you change names that follow the user settings: `oneTypo`, `twoTypos`, please?